### PR TITLE
ollama app welcome screen for first time run

### PR DESCRIPTION
**Pull request by hoyyeva**

### CODE REVIEW

File: app/assets/ollama.png
Comment (jmorganca) [inline app/assets/ollama.png:1]: Do we need both this and the svg?

File: app/forge.config.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const config: ForgeConfig = {
     new AutoUnpackNativesPlugin({}),
     new WebpackPlugin({
       mainConfig,
-      devContentSecurityPolicy: `default-src * 'unsafe-eval' 'unsafe-inline'`,
+      devContentSecurityPolicy: `default-src * 'unsafe-eval' 'unsafe-inline'; img-src data: 'self'`,
```
Comment (jmorganca) [inline app/forge.config.ts:61]: Does this work with `npm run package`? I think it would

File: app/src/app.tsx [hunk 2]
```diff
@@ -1,160 +1,129 @@
...
+  exec(`osascript -e '${command}'`, (error: Error | null, stdout: string, stderr: string) => {
+    if (error) {
+      console.error(`cli: failed to install cli: ${error.message}`)
+      callback && callback()
+      return
     }
-  }
 
-  return
+    console.info(stdout)
```
Comment (jmorganca) [inline app/src/app.tsx:32]: Remove me?

File: app/forge.config.ts (hunk 1 above)
Comment (hoyyeva) [reply app/forge.config.ts:61]: it does!

File: app/src/index.ts [hunk 3]
```diff
@@ -30,7 +33,37 @@ if (!SingleInstanceLock) {
...
-const createSystemtray = () => {
+
+function firstRunWindow() {
+  // Create the browser window.
+  welcomeWindow = new BrowserWindow({
+    width: 400,
+    height: 500,
+    frame: false,
+    fullscreenable: false,
+    resizable: false,
```
Comment (jmorganca) [inline app/src/index.ts:44]: nice!

**Review by jmorganca — APPROVED** [review]
lgtm!

**Pull request merged**